src/popup/components/Modal.vue

```diff
@@ -20,7 +20,10 @@
         <div
           v-if="showHeader"
           class="header"
-          :class="{ transparent: hasCloseButton && !($slots.header || header) }"
+          :class="{
+            transparent: hasCloseButton && !($slots.header || header),
+            'not-sticky-header': notStickyHeader,
```

Comment:
The header in my opinion should always be sticky as the close button should always be placed in the same top right corner. This class/prop is introducing kind of weird behavior.

If there is a need to place a header into modal it can be placed in the content slot.